### PR TITLE
DOCSUP-38271-pool-docs-update

### PR DESCRIPTION
**Pull request by lehasm**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation and examples improvement (changes affected documentation and/or examples)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/client/pool/speed_quality_balance_config.py [hunk 1]
```diff
@@ -1,38 +1,43 @@
...
+    Learn more about [Speed/quality balance](https://toloka.ai/en/docs/guide/concepts/adjust).
+    """
+
+    @unique
+    class Type(ExtendableStrEnum):
+        """The type of the filter used in [SpeedQualityBalanceConfig](toloka.client.pool.speed_quality_balance_config.SpeedQualityBalanceConfig.md).
+
+        Attributes:
+            TOP_PERCENTAGE_BY_QUALITY — A percentage of the best Tolokers is configured.
+            BEST_CONCURRENT_USERS_BY_QUALITY — A maximum number of the best Tolokers is configured.
```
Comment (alexdrydew) [inline src/client/pool/speed_quality_balance_config.py:26]: Сборка падает из-за тире тут: это не соответствует формату google-style докстрингов, должно быть двоеточие

File: src/client/__init__.py [hunk 2]
```diff
@@ -1389,49 +1396,49 @@ def clone_pool(self, pool_id: str) -> Pool:
...
 
         You can send a maximum of 20 requests of this kind per minute and 100 requests per day.
 
         Args:
-            pool: New Pool with set parameters.
+            pool: The pool to be created.
 
         Returns:
-            Pool: Created pool. With read-only fields.
+            Pool: The pool with updated read only parameters.
```
Comment (alexandervnuchkov) [inline src/client/__init__.py:1428]: `read-only` should be hyphenated (see https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/r/read-only)

File: src/client/pool/__init__.py [hunk 3]
```diff
@@ -35,113 +35,109 @@
...
+            Allowed range: from 0 to 259200 seconds (3 days). The default value is 0.
         dynamic_pricing_config: The dynamic pricing settings.
-        auto_accept_solutions: Whether tasks must be checked manually:
-            * True - Automatic task acceptance (manual checking isn't necessary).
-            * False - The requester will check the tasks.
-        auto_accept_period_day: Optional[int]
+        auto_accept_solutions:
+            * True — Responses from Tolokers are accepted or rejected automatically based on some rules.
+            * False — Responses are checked manually. Time reserved for checking is limited by the `auto_accept_period_day` parameter.
+                Learn more about [non-automatic acceptance](https://toloka.ai/en/docs/guide/concepts/offline-accept) in the Requester's guide.
```
Comment (alexandervnuchkov) [inline src/client/pool/__init__.py:74]: We no longer use the `Requester's guide` term in the docs. Please replace this sentence with `Learn more about [non-automatic acceptance](https://toloka.ai/en/docs/guide/concepts/offline-accept).`

File: src/client/pool/__init__.py [hunk 4]
```diff
@@ -35,113 +35,109 @@
...
-        last_started: The date and time when the pool was last started. Read only field.
-        last_stopped: The date and time when the pool was last stopped. Read only field.
-        type: Types of pool. Read only field.
+        speed_quality_balance: Settings for choosing Tolokers for your tasks.
+        dynamic_overlap_config: Dynamic overlap settings.
+        mixer_config: Parameters for automatically creating task suites.
+        training_config: Additional settings for linked training.
+        metadata: A dictionary with metadata.
+        owner: The pool owner.
+        id: The ID of the pool. Read only.
```
Comment (alexandervnuchkov) [inline src/client/pool/__init__.py:89]: See the remark above about `read-only` here and forth

File: src/client/pool/__init__.py [hunk 5]
```diff
@@ -35,113 +35,109 @@
...
-            COMPLETED: Closed automatically because all the pool tasks were completed.
-            NOT_ENOUGH_BALANCE: Closed automatically because the Toloka account ran out of funds.
-            ASSIGNMENTS_LIMIT_EXCEEDED: Closed automatically because it exceeded the limit on assigned task suites
-                (maximum of 2 million).
-            BLOCKED: Closed automatically because the requester's account was blocked by a Toloka administrator.
+            MANUAL: A pool was closed by a requester.
+            EXPIRED: The lifetime of the pool expired.
+            COMPLETED: All tasks were completed.
+            NOT_ENOUGH_BALANCE: There is not enough money to run the pool.
+            ASSIGNMENTS_LIMIT_EXCEEDED: A limit of 2 millions assignments is reached.
```
Comment (alexandervnuchkov) [inline src/client/pool/__init__.py:138]: Should be `2 million` as strange as it might seem

File: src/client/pool/mixer_config.py [hunk 6]
```diff
@@ -6,47 +6,46 @@
 
 
 class MixerConfig(BaseTolokaObject):
-    """Parameters for automatically creating a task suite ("smart mixing").
+    """Settings for automatically grouping tasks into suites (smart mixing).
 
-    For more information about creating task see Toloka Requester's guide.
+    You can set the number of general, training and control tasks in a task suite. Also you control task shuffling and other settings.
```
Comment (alexandervnuchkov) [inline src/client/pool/mixer_config.py:11]: `Also, you control task shuffling and other settings.`

File: src/client/pool/mixer_config.py [hunk 7]
```diff
@@ -6,47 +6,46 @@
...
+        min_golden_tasks_count: The minimum number of control tasks in a task suite if there are not enough control tasks left to create a full task suite.
+
+            Allowed range: from 0 to `golden_tasks_count`.
+            By default, the `min_golden_tasks_count` value equals to the `golden_tasks_count` value.
+        min_training_tasks_count: The minimum number of training tasks in a task suite if there are not enough training tasks left to create a full task suite.
+
+            Allowed range: from 0 to `training_tasks_count`.
+            By default, the `min_training_tasks_count` value equals to the `training_tasks_count` value.
+        force_last_assignment: A setting used when the number of remaining general tasks in the pool is less than the `min_real_tasks_count` value.
+            Note, that there must be enough control and trining tasks to create a task suite.
```
Comment (alexandervnuchkov) [inline src/client/pool/mixer_config.py:32]: `training` (typo)

**Review by alexandervnuchkov — CHANGES_REQUESTED** [review]

File: src/client/pool/speed_quality_balance_config.py (hunk 1 above)
Comment (lehasm) [reply src/client/pool/speed_quality_balance_config.py:26]: Thanks.

File: src/client/__init__.py (hunk 2 above)
Comment (lehasm) [reply src/client/__init__.py:1428]: Fixed _read-only_ in my PR.

There are examples without a hyphen in other files. I did not touch them.

**Review by alexdrydew — APPROVED** [review]

**Pull request merged**